### PR TITLE
Mighty game environment

### PR DESCRIPTION
**Pull request by ayo79**
add Mighty environment

### CODE REVIEW

**Comment by martinkersner** [comment]
@ayo79 thank you for **great** pull request! At the following weekend I will check it more in depth, but for now, could you please remove `*.xml` files generated by Visual Code Studio:

* .idea/vcs.xml
* seoulai_gym/.idea/misc.xml
* seoulai_gym/.idea/modules.xml
* seoulai_gym/.idea/workspace.xml

If you want to avoid adding them to repository next time, you can include them in `https://github.com/seoulai/gym/blob/master/.gitignore`.

Seoul AI gym is an open source project and we try to minimize differences between code from contributors. For this purpose we follow `PEP 8: Style Guide for Python Code` https://www.python.org/dev/peps/pep-0008. Every commit is checked against `flake8` https://pypi.org/project/flake8/ which helps us to follow this PEP. In your pull request there are several incompatibilities. Could you please fix them? After you push your changes, `flake8` test will run again and if everything is okay, you should see at your pull request `green check sign`.

Recently, I slightly updated gym interface because it didn't allow simple using of two different agents, but haven't updated master yet. After, I will help you to integrate to your code base as well.

If you have any questions, feel free to ask!

Cheers,
Martin

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (martinkersner) [inline .idea/vcs.xml]: Please remove this file

File: examples/mighty_example.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+"""
+Seung-Hyun Kim, kimseunghyun@gmail.com
+seoulai.com
+2018
+"""
+import seoulai_gym as gym
+#from seoulai_gym.envs.mighty.agents import RandomAgent, AlangAgent
```
Comment (martinkersner) [inline examples/mighty_example.py]: Please remove unused commented imports

File: seoulai_gym/.idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (martinkersner) [inline seoulai_gym/.idea/misc.xml]: Please remove this file

File: seoulai_gym/.idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (martinkersner) [inline seoulai_gym/.idea/modules.xml]: Please remove this file

File: seoulai_gym/.idea/seoulai_gym.iml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (martinkersner) [inline seoulai_gym/.idea/seoulai_gym.iml]: Please remove this file, it doesn't look necessary

File: seoulai_gym/.idea/workspace.xml [hunk 6]
```diff
@@ -0,0 +1,277 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (martinkersner) [inline seoulai_gym/.idea/workspace.xml]: Please remove this file

**Comment by ayo79** [comment]
OK. I can see which lines make problems. 
I will follow PEP 8: Style Guide for Python Code soon.

**Comment by ayo79** [comment]
Finally, I have passed all checks!

**Comment by martinkersner** [comment]
Awesome! I will check the rest tomorrow.

**Pull request merged**